python/lsst/meas/base/forcedMeasurement.py

```diff
@@ -51,17 +51,18 @@
 import lsst.pipe.base
 
 from .pluginRegistry import PluginRegistry
-from .baseMeasurement import BasePluginConfig, BasePlugin, BaseMeasurementConfig, BaseMeasurementTask
+from .baseMeasurement import BaseMeasurementPluginConfig, BaseMeasurementPlugin
+from .baseMeasurement import BaseMeasurementConfig, BaseMeasurementTask
```

Comment:
Why split this into two lines?  Is it to support a future change?  Or is it just because it's too long?  Suggest either one `import` per line (easy on git to add/subtract) or use parens around the list and just continue the line.